.github/ISSUE_TEMPLATE/add_package_request.md

```diff
@@ -19,10 +19,9 @@ List any other related pacakges already in hpc-stack, as far as you know.
 **Additional context**
 Add any other context or screenshots about the package or library here.
 
-**Checklist**
+**Will This Package be Needed in WCOSS Systems?**
```

Comment:
I think the appropriate question to ask is if this package will be a part of an operational application.  If yes, then please fill out as much information below.
If not, this package will not be made available on operational platforms e.g. WCOSS.

Reply:
OK...

Reply:
just brainstorming.